docs/content/latest/yugabyte-cloud/cloud-basics/connect/ycql.md

```diff
@@ -4,7 +4,7 @@ To connect to a cluster using `ycqlsh`:
 1. Click **Connect**.
 1. Click **YugabyteDB Client Shell**.
 1. If you have not installed the client shell on your computer, select your operating system and copy the command to install the shell from the command line.
-1. Click **Download CA Cert** to download the root.crt certificate for TLS encryption, and install the certificate on your computer.
+1. Click **Download CA Cert** to download the root.crt certificate for TLS encryption, and install the certificate on your computer. If you are using Docker, copy it to your Docker container.
```

Comment:
If you are using Docker, copy it to your Docker container.  -->  Reads like "it" refers to Docker. Should be changed to cert